pages/manage/productions/index.vue

```diff
@@ -1,47 +1,35 @@
 <template>
   <div class="wrapper">
-    <div>
+    <div class="container-fluid">
       <h1 class="title">
-       $t('menu.productions')
+        {{ $t('menu.productions') }}
+        <small>
```

Comment:
This link is still too big compared to other pages in the AngularJS app. It's now `19.2px`, but it should be `15.6px`. The `<small>` in the AngularJS app was overwritten to be font-size `65%` instead of the default `85%` in Bootstrap 3 (and `80%` in Bootstrap 4). So we should probably also change it to `65%`.